client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -897,10 +907,93 @@ class CancelPurchaseForm extends Component {
 		return parseFloat( refundAmount ).toFixed( precision );
 	};
 
+	renderBenefitsGrid = () => {
+		const { purchase, translate } = this.props;
+
+		switch ( purchase.productSlug ) {
+			case 'business-bundle':
+				return (
+					<div className={ 'cancel-purchase-form__benefits-wrapper' }>
+						<div className={ 'cancel-purchase-form__benefits-wrapper-grid' }>
+							{ /* TODO: translate the below */ }
+							<div>
+								{ /* would be good not to hardcode this */ }
```

Comment:
https://public-api.wordpress.com/rest/v1.5/plans will give a list of all possible plans, a individual plan will be structured like:


```{
	"25": {
		"product_id": 1028,
		"product_name": "WordPress.com Business",
		"meta": null,
		"bd_slug": "wp-bundles",
		"bd_variation_slug": "wp-bundle-business-2y",
		"sale_coupon_applied": false,
		"sale_coupon": null,
		"multi": 1,
		"cost": 399,
		"is_cost_from_introductory_offer": false,
		"blog_id": null,
		"bundle_product_ids": {
			"0": 12,
			"1": 45,
			"2": 15,
			"4": 50,
			"5": 49,
			"6": 5,
			"7": 20
		},
		"path_slug": "business-2-years",
		"product_slug": "business-bundle-2y",
		"description": "",
		"bill_period": 730,
		"product_type": "bundle",
		"available": "yes",
		"outer_slug": "",
		"extra": "",
		"capability": "manage_options",
		"product_name_short": "Business",
		"bill_period_label": "per year",
		"price": "£399",
		"formatted_price": "£399",
		"raw_price": 399,
		"tagline": null,
		"currency_code": "GBP"
	}
}
```

`bundle_product_ids` contains ids of bundled products, a product is not just "storage" but "200gb storage", "1gb storage" etc. 

In future, possibly people could buy unbundled storage space products. 

Relying on the plan alone to determine their current and future storage space is unreliable and in some cases possibly not a compelling sell.